db/migrate/20240306121718_add_displayed_comments_count_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisplayedCommentsCountToArticles < ActiveRecord::Migration[7.0]
+  def change
+    add_column :articles, :displayed_comments_count, :integer
```

Comment:
should we add a default here to 0?

Reply:
`nil` means that the count has not been calculated yet, the display logic is based on this: in the feed we display `comments_count` when `displayed_comments_count` is `nil`, and on the show page we calculate `displayed_comments_count`. If default is nil we can't figure out if it's actually nil or hasn't been calculated yet.